app/scripts/modules/core/src/search/searchResult/searchResults.less

```diff
@@ -16,18 +19,16 @@
     width: 275px;
 
     .search-group {
+      > * {
+        flex: 0 0 auto;
+      }
 
       align-items: baseline;
       .clickable();
       display: flex;
```

Comment:
May be use the existing selectors here too? I believe there is .horizontal / .vertical and flex-v?